.npmrc

```diff
@@ -0,0 +1,2 @@
+save-exact=true
+update-notifier=false
```

Comment:
Let's keep a new line at the end of all files, as mentioned in the POSIX standard.